libs/live-network/src/network.ts

```diff
@@ -168,6 +168,12 @@ const implementation = <T = any>(arg: AxiosRequestConfig): AxiosPromise<T> => {
 
     promise = retry(() => axios(arg), {
       maxRetry: getEnv("GET_CALLS_RETRY"),
+      retryCondition: error => {
+        if (error && error.status) {
+          return error.status !== 422;
+        }
+        return true;
+      },
```

Comment:
As a general comment, I feel like this a very specific to a single endpoint behaviour ? I'm not super familiar with 422 errors, but it feels to me like there is no specific reason to not retry on this more than on 404 for example. 
Couldn't we add that `retryCondition` as an optional argument of `network` so that this can be adapted to the different responses possible for a request ?

Reply:
As per [this documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422), a 422 Unprocessable content shouldn't be retried without modification so it makes sense to me to make it general and not optional. What do you think ?

Reply:
Ah, interesting ! I'm more blind everyday and I didn't notice the big red warning 😆
By reading it I see that there is still an option to retry it based on a response header: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Retry-After (which I guess would make the retryCondition way harder to use, so just mentioning it as I think it's weird to document "you shouldn't retry this as is. Except after this date. 🤔)

I'm still trying to get my head around a 422 is less "retriable" than a 404 but I guess that's another question. 

Can you please just add a comment explaining that 422 are documented this way and link the doc ? Just to make sure someone else will understand this without context and not brain fart like I did 👍